db/src/models/users.rs

```diff
@@ -700,101 +699,63 @@ impl User {
+                    WHERE COALESCE(o.on_behalf_of_user_id, o.user_id) = $1
+                    AND e.organization_id = $2
                     AND o.status = 'Paid'
                     AND oi.item_type = 'Tickets'
-                ), 0) as BigInt)",
-                ),
-                sql::<BigInt>(
-                    "CAST(COALESCE((
+                ), 0) as BigInt) as ticket_sales,
+                    CAST(COALESCE((
                     SELECT COUNT(ti.id)
                     FROM ticket_instances ti
                     JOIN wallets w ON w.id = ti.wallet_id
                     JOIN assets a on ti.asset_id = a.id
                     JOIN ticket_types tt on tt.id = a.ticket_type_id
                     JOIN events e ON e.id = tt.event_id
-                    WHERE w.user_id = users.id
-                    AND e.organization_id = events.organization_id
+                    WHERE w.user_id = $1
+                    AND e.organization_id = $2
                     AND ti.status in ('Purchased', 'Redeemed')
-                ), 0) as BigInt)",
-                ),
-                sql::<BigInt>(
-                    "CAST(COALESCE((
+                ), 0) as BigInt) as tickets_owned,
+                 CAST(COALESCE((
                     SELECT SUM(oi.unit_price_in_cents * (oi.quantity - oi.refunded_quantity))
                     FROM order_items oi
                     JOIN orders o ON o.id = oi.order_id
                     JOIN events e ON e.id = oi.event_id
-                    WHERE COALESCE(o.on_behalf_of_user_id, o.user_id) = users.id
-                    AND e.organization_id = events.organization_id
+                    WHERE COALESCE(o.on_behalf_of_user_id, o.user_id) = $1
+                    AND e.organization_id = $2
                     AND o.status = 'Paid'
-                ), 0) as BigInt)",
-                ),
-                sql::<BigInt>("cast(COALESCE(count(distinct events.id), 0) as BigInt)"),
-            ));
+                ), 0) as BigInt) as revenue_in_cents",
+        )
+        .bind::<dUuid, _>(self.id)
+        .bind::<dUuid, _>(organization.id);
 
-        #[derive(Queryable)]
+        #[derive(QueryableByName)]
         struct R {
+            #[sql_type = "BigInt"]
             ticket_sales: i64,
+            #[sql_type = "BigInt"]
             tickets_owned: i64,
+            #[sql_type = "BigInt"]
             revenue_in_cents: i64,
-            event_count: i64,
         }
-        let result: R = query
-            .get_result(conn)
+        let mut result: Vec<R> = query
+            .get_results(conn)
             .to_db_error(ErrorCode::QueryError, "Could not load profile for organization fan")?;
 
+        let result = result.remove(0);
+        if result.ticket_sales == 0 && result.tickets_owned == 0 && result.revenue_in_cents == 0 {
+            return Err(DatabaseError::no_results("No profile information for this organization"));
```

Comment:
The DieselError::NotFound appends the `, NotFound` into the error message.
```suggestion
            return Err(DatabaseError::no_results("No profile information for this organization, NotFound"));
```